docsrc/tutorials/getting_started.rst

```diff
@@ -138,20 +147,23 @@ to load in a deployment application. In order to load a TensorRT/TorchScript mod
     import trtorch
 
     ...
+
+    script_model.eval() # torch module needs to be in eval (not training) mode
+
     compile_settings = {
         "input_shapes": [
             {
-                "min": [1, 3, 224, 224],
-                "opt": [1, 3, 512, 512],
-                "max": [1, 3, 1024, 1024]
-            }, # For static size [1, 3, 224, 224]
+                "min": [1, 1, 32, 32],
```

Comment:
Lets vary the sizes here, it is supposed to demonstrate dynamic shape

Reply:
done